build_aarch64_wheel.py

```diff
@@ -50,7 +58,16 @@ def start_instance(key_name, ami=ubuntu18_04_ami, instance_type='t4g.2xlarge'):
                                 SecurityGroups=['ssh-allworld'],
                                 KeyName=key_name,
                                 MinCount=1,
-                                MaxCount=1)[0]
+                                MaxCount=1,
+                                BlockDeviceMappings=[
```

Comment:
Can we make it an option?
Also, are volumes automatically destroyed when terminate instances is called?